tgui/packages/tgui/interfaces/CargoHoldTerminal.js

```diff
@@ -0,0 +1,54 @@
+import { Fragment } from 'inferno';
+import { useBackend } from '../backend';
+import { AnimatedNumber, Button, Box, Section, LabeledList } from '../components';
+
+export const CargoHoldTerminal = props => {
+  const { act, data } = useBackend(props);
+  const {
+    points,
+    pad,
+    sending,
+    status_report,
+  } = data;
+  return (
+    <Fragment>
+      <Section>
+        <LabeledList>
+          <LabeledList.Item label="Current Cargo Value">
+            <Box inline bold>
+              <AnimatedNumber value={Math.round(points)} /> credits
+            </Box>
+          </LabeledList.Item>
+        </LabeledList>
+      </Section>
+      <Section
+        title="Cargo Pad"
+        buttons={(
+          <Fragment>
+            <Button
+              icon={"sync"}
+              content={"Recalculate Value"}
+              disabled={!pad}
+              onClick={() => act('recalc')} />
+            <Button
+              icon={sending ? 'times' : 'arrow-up'}
+              content={sending ? "Stop Sending" : "Send Goods"}
+              selected={sending}
+              disabled={!pad}
+              onClick={() => act(sending ? 'stop' : 'send')} />
+          </Fragment>
+        )}>
+        <LabeledList>
+          <LabeledList.Item
+            label="Status"
+            color={pad ? "good" : "bad"}>
+            {pad ? "Online" : "Not Found"}
+          </LabeledList.Item>
+          <LabeledList.Item label="Cargo Report">
```

Comment:
```
style={{
  'white-space': 'pre',
}}
```
To preserve line breaks.

Reply:
I could not get that style working on `<LabeledList.Item>` or `<LabelledList>`, or any style for that matter. I could however put the `{status_report}` in `<Box>` and then style the box with that, but that resulted in that style acting like `white-space : nowrap`, with the entire report on one line. No idea how to move further with that.

Reply:
Sounds like a table-related quirk, because LabeledList is a table under the hood. And that just makes me sad.